server/home/homeDB.js

```diff
@@ -0,0 +1,20 @@
+
+const {Movie} = require('../database/database.js');
+const {transformToHomeResponse} = require('./movieHelpers')
+// console.log(redisClient)
+module.exports = {
+  getMovie:  (movieId) => {
+    //if there is movie id, we know it exists in DB - so lets query it 🔥
+    if (movieId) {
```

Comment:
is this where the logic will go for referring to the api if there is no movie id?

Reply:
home will always have an id to search by.  That logic will be happening in the search component